R/system.R

```diff
@@ -1208,7 +1208,7 @@ getGoogleBigQueryTables <- function(project, dataset, tokenFileId=""){
     token <- getGoogleTokenForBigQuery(tokenFileId);
     bigrquery::set_access_cred(token)
     # if we do not pass max_results, it only returnss 50 items. so explicitly set it.
```

Comment:
Does max_results here mean page_size?
Also, please add comment on what you found about the error you are fixing in this PR.

Reply:
I added a comment about the error I got from Google BigQuery API call and my observation.